src/components/views/rooms/RoomHeader.tsx

```diff
@@ -104,6 +116,95 @@ export default function RoomHeader({
 
     const askToJoinEnabled = useFeatureEnabled("feature_ask_to_join");
 
+    const toggleCallButton = (
+        <Tooltip label={isViewingCall ? _t("voip|minimise_call") : _t("voip|maximise_call")}>
+            <IconButton onClick={toggleCall}>
+                <VideoCallIcon />
+            </IconButton>
+        </Tooltip>
+    );
+    const joinCallButton = (
+        <Button
+            size="sm"
+            onClick={(ev) => videoCallClick(ev, callOptions[0])}
+            Icon={VideoCallIcon}
+            className="mx_RoomHeader_join_button"
+            color="primary"
+        >
+            {_t("action|join")}
+        </Button>
+    );
+    const [menuOpen, setMenuOpen] = useState(false);
+    const callIconWithTooltip = (
+        <Tooltip label={videoCallDisabledReason ?? _t("voip|video_call")}>
+            <VideoCallIcon />
+        </Tooltip>
+    );
+    const startVideoCallButton = (
+        <>
+            {/* Can be either a menu or just a button depending on the number of call options.*/}
+            {callOptions.length > 1 ? (
+                <Menu
+                    open={menuOpen}
+                    onOpenChange={setMenuOpen}
+                    title="Video call using:"
+                    trigger={
+                        <IconButton
+                            disabled={!!videoCallDisabledReason}
+                            aria-label={videoCallDisabledReason ?? _t("voip|video_call")}
+                        >
+                            {callIconWithTooltip}
+                        </IconButton>
+                    }
+                    side="left"
+                    align="start"
+                >
+                    {callOptions.map((option) => (
+                        <MenuItem
+                            key={option}
+                            label={getPlatformCallTypeLabel(option)}
+                            onClick={(ev) => videoCallClick(ev, option)}
+                            Icon={VideoCallIcon}
+                            onSelect={() => {} /* Dummy handler since we want the click event.*/}
```

Comment:
Does this still respond to selecting the menu item with the Enter key?

Reply:
I expected it to not do so and implemnetd both with appropriate conditions for mouse events. But that resulted in bugs becuase videoCallClick was called twice.

So this definitly works with only implmeneting onClick. (could this be considered as a bug in the menu component?)